<h2>Eye conditions and diseases among people</h2>
<br></br>
<h3>Introduction</h3>
<p>Eye-related conditions and diseases encompass a wide range of disorders that affect vision and overall eye health. These conditions, from common issues like refractive errors and cataracts to complex diseases such as glaucoma and age-related macular degeneration, can significantly impact quality of life. Early detection, diagnosis, and advancements in research are crucial for effective treatment and prevention, highlighting the importance of continuous study in the field of ophthalmology.</p>
<br></br>
<h3>Data set</h3>

In [1]:
from pandas import read_csv
data = read_csv('participants_info.csv')
data

id_record        date  age_years     sex  \
0            1  2016-09-15         13    Male   
1            2  2005-09-15         13  Female   
2            3  2019-08-08         49  Female   
3            4  2004-12-16         43  Female   
4            5  2016-07-13         47  Female   
..         ...         ...        ...     ...   
331        332  2016-11-17         43    Male   
332        333  2015-01-14          4    Male   
333        334  2021-08-28         14    Male   
334        335  2007-12-20         23  Female   
335        336  2003-12-04          8  Female   

                                diagnosis1  \
0                                   Normal   
1    Congenital stationary night blindness   
2                         Orbital ischemia   
3                     Retinitis pigmentosa   
4                                   Normal   
..                                     ...   
331                 Autoimmune retinopathy   
332                 X linked retinoschisis   
333                                 Normal   
334                   Retinitis pigmentosa   
335                                 Normal   

                                       diagnosis2 diagnosis3  va_re_logMar  \
0                                             NaN        NaN         -0.08   
1                                             NaN        NaN          0.18   
2    Systemic disorder with ocular manifestations        NaN          0.26   
3                                             NaN        NaN           NaN   
4                                             NaN        NaN          0.10   
..                                            ...        ...           ...   
331                          Inflammatory disease        NaN          0.02   
332                                           NaN        NaN          1.00   
333                                           NaN        NaN         -0.10   
334                                           NaN        NaN          0.00   
335                                           NaN        NaN          0.00   

     va_le_logMar unilateral                             rep_record  \
0            0.06        NaN                                    NaN   
1            0.16        NaN                                    NaN   
2            0.00        NaN  Id:0329 - Id:0154 - Id:0049 - Id:0271   
3             NaN        NaN                                    NaN   
4            0.10        NaN                                    NaN   
..            ...        ...                                    ...   
331          0.00        NaN            Id:0221 - Id:0252 - Id:0126   
332          0.52        NaN                                    NaN   
333         -0.06        NaN                                    NaN   
334          0.00        NaN                                    NaN   
335          0.00        NaN                                Id:0196   

                                comments  
0                                    NaN  
1                                    NaN  
2                                    NaN  
3                                    NaN  
4                                    NaN  
..                                   ...  
331                                  NaN  
332                                  NaN  
333  Family history of chorioretinopathy  
334                                  NaN  
335                                  NaN  

[336 rows x 12 columns]

<h2>Comments on data before describing it</h2>
<p>The data provides a very wide information on observations upon people's eyesight, and this gives us a plenty of interpretations we can extract from the set.</p>
<br></br>
<h2>Describing data</h2>
<h4>Population</h4>
<p>The participants are people from  Valladolid, Spain, who participated in the research from 2003 to 2022</p>
<h4>Size</h4>
<p>336 people</p>
<h4>Data description</h4>
<p>The data gives us a lot of information from sex and age to the number of visits. However, I am mainly going to focus on the age of the participant, their sex and whether their eyesight is normal or they have a certain number of diseases.</p>
<h4>Variables:</h4>
<h5>Variable AGE</h5>
<p>Variable name: AGE</p>
<p>Variable description: the variable is the age of a participant</p>
<p>Variable type: continuous</p>
<p>Range: [0,100]</p>
<h5>Variable SEX</h5>
<p>Variable name: SEX</p>
<p>Variable description: the variable is the sex of a participant</p>
<p>Variable type: categorical</p>
<p>Range: [Male,Female]</p>
<h5>Variable DISEASES</h5>
<p>Variable name: DISEASES</p>
<p>Variable description: the variable is the number of diseases a participant has</p>
<p>Variable type: discrete</p>
<p>Range: [0,3]</p>
<br></br>
<h2>Why is the data relevant?</h2>
<p>The set has many observations (300+). The set has a lot of columns, so it has much statistical series, which helps me identify the information I am going to extract clearly. The records were produced by the University of Valladolid, so I am pretty sure I can trust the authors. In my opinion, all these make the conditions to observe the data sufficient.</p>
<br></br>
<h2>Flat view of the first lines (displayed below for convenience)</h2>

In [2]:
data[:10]

id_record        date  age_years     sex  \
0          1  2016-09-15         13    Male   
1          2  2005-09-15         13  Female   
2          3  2019-08-08         49  Female   
3          4  2004-12-16         43  Female   
4          5  2016-07-13         47  Female   
5          6  2006-10-07         20    Male   
6          7  2015-04-10         43    Male   
7          8  2007-01-24         10  Female   
8          9  2018-04-12         13    Male   
9         10  2007-03-24         35  Female   

                              diagnosis1  \
0                                 Normal   
1  Congenital stationary night blindness   
2                       Orbital ischemia   
3                   Retinitis pigmentosa   
4                                 Normal   
5                       Retinal toxicity   
6                                 Normal   
7                      Macular dystrophy   
8                Inherited optic atrophy   
9                   Retinitis pigmentosa   

                                     diagnosis2             diagnosis3  \
0                                           NaN                    NaN   
1                                           NaN                    NaN   
2  Systemic disorder with ocular manifestations                    NaN   
3                                           NaN                    NaN   
4                                           NaN                    NaN   
5                                           NaN                    NaN   
6                                           NaN                    NaN   
7                             Stargardt disease  Fundus flavimaculatus   
8                                           NaN                    NaN   
9                                           NaN                    NaN   

   va_re_logMar  va_le_logMar unilateral  \
0         -0.08          0.06        NaN   
1          0.18          0.16        NaN   
2          0.26          0.00        NaN   
3           NaN           NaN        NaN   
4          0.10          0.10        NaN   
5         -0.06          0.04        NaN   
6         -0.10          0.07        NaN   
7          1.00          0.98        NaN   
8          0.32          0.26        NaN   
9         -0.07          0.07        NaN   

                              rep_record                           comments  
0                                    NaN                                NaN  
1                                    NaN                                NaN  
2  Id:0329 - Id:0154 - Id:0049 - Id:0271                                NaN  
3                                    NaN                                NaN  
4                                    NaN                                NaN  
5                                Id:0112  Toxic maculopathy by isotretinoin  
6                                    NaN                  Mercury poisoning  
7                                    NaN                                NaN  
8                                    NaN                           Dominant  
9                                    NaN                                NaN

<p>As we see from the data, we have participants of different age and sex. Some of them have a normal eyesight, some have up to 3 diseases. The data is not processed, and I cannot make any conclusions for the time now, but I see that the observations are satisfying in terms of information.</p>